ui/apps/platform/src/Containers/NetworkGraph/externalEntities/ExternalEntitiesSideBar.tsx

```diff
@@ -33,80 +38,26 @@ type ExternalEntitiesSideBarProps = {
     edges: CustomEdgeModel[];
 };
 
-const flows: Flow[] = [
-    {
-        id: 'Deployment 1-naples-Ingress-Many-TCP',
-        type: 'Deployment',
-        entity: 'Deployment 1',
-        namespace: 'naples',
-        direction: 'Bi-directional',
-        port: 'MANY',
-        protocol: 'TCP',
-        isAnomalous: true,
-        children: [
-            {
-                id: 'Deployment 1-naples-Ingress-Many-TCP',
-                type: 'Deployment',
-                entity: 'Deployment 1',
-                namespace: 'naples',
-                direction: 'Egress',
-                port: '9000',
-                protocol: 'TCP',
-                isAnomalous: true,
-            },
-            {
-                id: 'Deployment 1-naples-Ingress-Many-TCP',
-                type: 'Deployment',
-                entity: 'Deployment 1',
-                namespace: 'naples',
-                direction: 'Ingress',
-                port: '8000',
-                protocol: 'TCP',
-                isAnomalous: true,
-            },
-        ],
-    },
-    {
-        id: 'Deployment 2-naples-Ingress-Many-UDP',
-        type: 'Deployment',
-        entity: 'Deployment 2',
-        namespace: 'naples',
-        direction: 'Ingress',
-        port: '8080',
-        protocol: 'UDP',
-        isAnomalous: true,
-        children: [],
-    },
-    {
-        id: 'Deployment 3-naples-Egress-7777-UDP',
-        type: 'Deployment',
-        entity: 'Deployment 3',
-        namespace: 'naples',
-        direction: 'Egress',
-        port: '7777',
-        protocol: 'UDP',
-        isAnomalous: true,
-        children: [],
-    },
-];
-
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 function ExternalEntitiesSideBar({ id, nodes, edges }: ExternalEntitiesSideBarProps): ReactElement {
+    const controller = useVisualizationController();
     // component state
     const [entityNameFilter, setEntityNameFilter] = React.useState<string>('');
     const [advancedFilters, setAdvancedFilters] = React.useState<AdvancedFlowsFilterType>(
         defaultAdvancedFlowsFilters
     );
-    const initialExpandedRows = flows
+    const flows = getNetworkFlows(edges, controller, id);
```

Comment:
oh hm idk how i feel about passing the whole controller through, but we can do this for now and iterate on it later